integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017-Present Okta, Inc.
+ * Copyright 2018-Present Okta, Inc.
```

Comment:
Revert the date changes to the licnese headers

Reply:
done